x/bank/simulation/operations_test.go

```diff
@@ -137,6 +196,33 @@ func (suite *SimTestSuite) getTestingAccounts(r *rand.Rand, n int) []simtypes.Ac
 	return accounts
 }
 
+func (suite *SimTestSuite) setupAccounts(moduleAccCount, accCount int) []simtypes.Account {
+	s := rand.NewSource(int64(moduleAccCount))
```

Comment:
we should use a more variable seed. Eg, timestamp

Reply:
u mean unix timestamp?